xml/System/ReadOnlySpan`1.xml

```diff
@@ -217,7 +217,12 @@ A `ReadOnlySpan<T>` instance is often used to reference the elements of an array
         <param name="length">The number of <typeparamref name="T" /> elements to be included in the <see cref="T:System.ReadOnlySpan`1" />.</param>
         <summary>Creates a new <see cref="T:System.ReadOnlySpan`1" /> from a specified number of <typeparamref name="T" /> elements starting at a specified memory address.</summary>
         <remarks>
-          <format type="text/markdown"><![CDATA[This constructor should be used with care, since it creates arbitrarily typed `T`s from a block of memory.]]></format>
+          <format type="text/markdown"><![CDATA[
+
+          This constructor should be used with care, since it creates arbitrarily typed `T`s from a `void*`-typed block of memory, and neither `pointer` nor `length` are validated.
+          However, if `pointer` and `length` point to a valid object, all subsequent uses are correct.
```

Comment:
They need to remain valid as well, e.g. if pointer points to a non-pinned object that's then moved by the GC, it's no longer valid. In such a case, that's a bug in the consumption of this ctor, which needs to ensure that the pointer is and remains valid as long as the resulting span might be used.

Reply:
(I would just delete the second sentence here.)